android/app/src/main/java/labs/pooh/eaterslab/util/Drawable.kt

```diff
@@ -16,5 +16,5 @@ fun convertDrawableToBitmap(context: Context, drawableId: Int): Bitmap? {
         drawable.setBounds(0, 0, canvas.width, canvas.height)
         drawable.draw(canvas)
         bitmap
-    }
+    }!!
```

Comment:
Is this correct syntax? `!!` sometimes means "cast to boolean", but this function returns a Bitmap.

Reply:
here it means cast to Bitmap insted of nullable Bitmap (kotlin syntax). It could be nullable anly in cases when there is valild context resource or something like that but it doesn''t happen here (as we have all needed resources) so nullable type is casted to non nulllable for simplicity